config/locales/en/results.yml

```
@@ -68,7 +68,18 @@ en:
         youth_simple_caution: Youth Simple caution
         youth_conditional_caution: Youth Conditional caution
 
+
   results/conviction:
+    conviction_title:
+      past: Your conviction was spent on
```

```suggestion
      past: Your conviction was spent on %{date}
```

(same for `present` sentence)